msm/src/main.rs

```diff
@@ -1,7 +1,10 @@
-use ark_ff::UniformRand;
```

Comment:
As column.rs, let's move this in a subdirectory for ffadd/ffmul in the next PR. Would be nice to clean this